examples/customizable/public/main.js

```diff
@@ -58,29 +39,14 @@ colorInput.addEventListener('input', () => {
 	colorInput.value = color = '#' + color.substr(1).replace(/[^a-f\d]/gi, '')
 
 	// Check if color is valid hex format
-	if (!/^#[a-f\d]{8}$/i.test(color)) {
-		colorInput.invalid = true
-		toggleAllowUpdate(false)
-		color = null
-		return
-	} else {
-		colorInput.invalid = false
-	}
-
-	toggleAllowUpdate(true)
+	colorInput.invalid = !/^#[a-f\d]{8}$/i.test(color)
 })
 
 function update(sync) {
-	if (!updateAllowed)
-		return
+	if (!updateAllowed) return
```

Comment:
Please delete this, too.

Reply:
Yep, forgot that. This is the perfect example why to use TypeScript. I settled with JavaScript anyways to keep a simple project environment.

Reply:
I tested the example after the fix again, and it worked like a charm.